lib/repositories/featured_collections_repository.dart

```diff
@@ -2,14 +2,14 @@ import 'dart:convert' as convert;
 import 'dart:io';
 
 import 'package:brush_strokes/env/env.dart';
-import 'package:brush_strokes/base/const.dart';
+import 'package:brush_strokes/const.dart';
 import 'package:brush_strokes/models/collections/featured_collections.dart';
 import 'package:http/http.dart' as http;
 
 class FeaturedCollectionsRepository {
   Future<FeaturedCollections> getFeaturedCollections() async {
     const featuredCollectionsEndpoint =
-        '$BASE_URL/collections/featured?per_page=5';
+        '$BASE_URL/collections/featured?per_page=10';
     final response = await http.get(
```

Comment:
There are a few issues with making API calls directly in repos:
1. API key logic is duplicated everywhere. 
2. Mixing domain (repository) and data source (API client or a database) layers. Repository should be more abstract and not depend on a particular API client implementation.

The better solution would be to have something like `PexelsApiClient` and then inject it. `PexelsApiClient` should be responsible for making REST calls and handle API key logic.

Reply:
See an example [here](https://bloclibrary.dev/#/flutterweathertutorial?id=openmeteo-api-client-1) ℹ️